src/game/behaviors/goomba.inc.c

```diff
@@ -285,7 +285,7 @@ static void floomba_act_startup(void) {
     
     cur_obj_unhide();
 
-    if ((GET_BPARAM3(o->oBehParams) & 0x7F) > o->oZoomCounter) {
+    if ((s32) (GET_BPARAM3(o->oBehParams) & 0x7F) > o->oZoomCounter) {
```

Comment:
incredibly useless cast, the & 0x7F would always return an unsigned number anyway

Reply:
This throws a compiler warning without it, it's either cast that or cast o->oZoomCounter

Reply:
shouldn't oZoomCounter be a u32 anyway?

Reply:
Fair point